src/Plugin/HistoryPlugin.php

```diff
@@ -40,7 +39,7 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
             $journal->addSuccess($request, $response);
 
             return $response;
-        }, function (Exception $exception) use ($request, $journal) {
+        }, function (\Exception $exception) use ($request, $journal) {
```

Comment:
Are we happy with the Journal only accepting HTTPlug exceptions? Or should we try to find an other solution?